test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -54,25 +49,22 @@ const clusterLabel = nscontroller.ClusterLabel
 func TestNamespaceScheduler(t *testing.T) {
 	t.Parallel()
 
-	const (
-		serverName           = "main"
-		physicalCluster1Name = "pcluster1"
-	)
-
 	type runningServer struct {
 		framework.RunningServer
-		client        kubernetes.Interface
-		clusterClient v1alpha1.WorkloadClusterInterface
-		expect        registerNamespaceExpectation
+		client           kubernetes.Interface
+		orgKcpClient     versioned.Interface
+		kcpClusterClient clientset.Interface
```

Comment:
this does not look like a cluster client

Reply:
Done